src/html/authorize.html.template

```diff
@@ -33,7 +33,7 @@
   <body>
     <div class="content">
       <div class="header">
-        <img src="/static/icons/favicon-192x192.png" height="52" />
+        <img src="/static/icons/favicon-192x192.png" height="52" aria-hidden="true" />
```

Comment:
Why would this be hidden?

Reply:
The image is right next to the "Home Assistant" header text, so giving the image an alt text of "Home Assistant" or "Home Assistant logo" I think would be repetitive. However, in retrospect I realize giving the image a null alt-text is the most common way [1] of handling this case, so I'm going to amend the PR.

[1] https://www.w3.org/WAI/tutorials/images/decorative/